<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainFrame</class>
 <widget class="QMainWindow" name="mainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LanChat</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../home/flyinbuff/res.qrc">
    <normaloff>:/icons/lanchat.ico</normaloff>:/icons/lanchat.ico</iconset>
  </property>
  <widget class="QWidget" name="mainLayout">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="chatTitle">
      <property name="minimumSize">
       <size>
        <width>622</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>15</height>
       </size>
      </property>
      <property name="text">
       <string>Chat with %user</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="chatWindow">
      <property name="minimumSize">
       <size>
        <width>622</width>
        <height>344</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="acceptRichText">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
      <property name="openLinks">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sendText">
      <property name="minimumSize">
       <size>
        <width>622</width>
        <height>27</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>27</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="minimumSize">
         <size>
          <width>114</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>114</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Files...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="downloadFrame" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="downloadLabel">
           <property name="text">
            <string>Downloading</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="downloadProgress">
           <property name="value">
            <number>24</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>downloadProgress</zorder>
        <zorder>downloadLabel</zorder>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="uploadFrame" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="uploadLabel">
           <property name="text">
            <string>Uploading</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="uploadProgress">
           <property name="value">
            <number>24</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../../home/flyinbuff/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sendText</sender>
   <signal>returnPressed()</signal>
   <receiver>sendButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
